source/ftrack_connect_pipeline/constants/asset/__init__.py

```diff
@@ -68,6 +68,18 @@
 #: :class:`~ftrack_connect_pipeline.asset.asset_info.FtrackAssetInfo`
 #: and the DCC ftrack plugin.
 OBJECTS_LOADED = 'objects_loaded'
+#: Determine if asset info is from a snapshot asset like an uasset. False by default.
+#: :class:`~ftrack_connect_pipeline.asset.asset_info.FtrackAssetInfo`
+#: and the DCC ftrack plugin.
+IS_SNAPSHOT = 'is_snapshot'
+#: Modification time of the file.
+#: :class:`~ftrack_connect_pipeline.asset.asset_info.FtrackAssetInfo`
+#: and the DCC ftrack plugin.
+MOD_DATETIME = 'mod_datetime'
```

Comment:
I would maybe just call this _mod_date_, so user does not think it's stored as a datetime Python object (getmtime returns seconds since epoch)